src/sos/singularity/client.py

```diff
@@ -350,10 +345,10 @@ def run(self,
 
             if ret != 0:
                 debug_script_dir = env.exec_dir
-                msg = 'The script has been saved to {}/{}. To reproduce the error please run:\n``{}``'.format(
-                    debug_script_dir, tempscript,
-                    cmd.replace(f'{path(tempdir):p}',
-                                f'{path(debug_script_dir):p}'))
+                msg = (
+                    f"The script has been saved to {debug_script_dir}/{debug_script_dir}."
+                    f"To reproduce the error please run:\n``{cmd.replace(f'{path(tempdir):p}', f'{path(debug_script_dir):p}')}``"
+                )
```

Comment:
Please define a variable for 
```cmd.replace(f'{path(tempdir):p}',
                                f'{path(debug_script_dir):p}')```

to avoid using nested f-string.